appveyor.yml

```diff
@@ -101,6 +101,9 @@ install:
   - "%CMD_IN_ENV% python setup.py bdist_wheel"
   - ps: "ls dist"
 
+  # Check for README compatibility in case we deploy
+  - "%CMD_IN_ENV% python -m twine check dist/*"
```

Comment:
If this doesn't pass, will the CI job fail?

Reply:
(it should, probably)

Reply:
It does not at the moment. When we hit this line with the old `setuptools` version, it gave a warning and continued. I’ll figure out a way to make the job fail if this step fails

Reply:
This will fail on improper rendering now